Cargo.toml

```diff
@@ -16,6 +16,7 @@ ruma = { version = "0.0.2", features = ["ruma-api"] }
 #js_int = { version = "0.2.0", features = ["serde"] }
 serde = { version = "1.0.118", features = ["derive"] }
 #serde_json = "1.0.61"
+ruma-serde = { version = "0.3.0" }
```

Comment:
You shouldn't have to add a direct dependency for this, it should be available as `ruma::serde`.